plugins/Lb302/Lb302.cpp

```diff
@@ -28,92 +28,98 @@
  *
  */
 
-// Need to include this first to ensure we get M_PI in MinGW with C++11
-#define _USE_MATH_DEFINES
+#include "Lb302.h"
+
 #include <cmath>
+#include <numbers>
+
+#include <QDebug>
 
-#include "Lb302.h"
 #include "AutomatableButton.h"
+#include "BandLimitedWave.h"
 #include "DspEffectLibrary.h"
 #include "Engine.h"
+#include "embed.h"
 #include "InstrumentPlayHandle.h"
 #include "InstrumentTrack.h"
 #include "Knob.h"
 #include "LedCheckBox.h"
 #include "NotePlayHandle.h"
 #include "Oscillator.h"
 #include "PixmapButton.h"
-#include "BandLimitedWave.h"
-
-#include "embed.h"
 #include "plugin_export.h"
 
-// Envelope Recalculation period
-#define ENVINC 64
-
-//
-// New config
-//
 #define LB_24_IGNORE_ENVELOPE
-#define LB_FILTERED
-//#define LB_DECAY
 //#define LB_24_RES_TRICK
```

Comment:
Since we cleaning up the codebase, it would make sense to remove this commented macro.